### PR TITLE
ER-1219: Added cateory list paragraphs type and view mode

### PR DESCRIPTION
**Pull request by rimi-itk**
https://jira.itkdev.dk/browse/ER-1219

### CODE REVIEW

File: ereol_inspiration/ereol_inspiration.features.field_instance.inc [hunk 1]
```diff
@@ -317,7 +317,7 @@ function ereol_inspiration_field_default_field_instances() {
   t('Categories');
   t('Category query');
   t('Elementer');
```
Comment (Anna-itk) [inline ereol_inspiration/ereol_inspiration.features.field_instance.inc:319]: Should 'Elementer' not be listed as 'Elements' as well?

**Review by Anna-itk — CHANGES_REQUESTED** [review]
A small question but everything else looks fine.

File: ereol_inspiration/ereol_inspiration.features.field_instance.inc (hunk 1 above)
Comment (rimi-itk) [reply ereol_inspiration/ereol_inspiration.features.field_instance.inc:319]: Yes, but that error has not been introduced in this pull request (cf. https://github.com/eReolen/ereolen/blame/0ef35d53b2ccf131eb587429728ff0b6f8bf5afb/ereol_inspiration/ereol_inspiration.features.field_instance.inc#L319).

**Review by Anna-itk — APPROVED** [review]
'Elementer' is an old missing translation we've agreed not to fix now. PR approved

**Pull request merged**